swarm/network/kademlia.go

```diff
@@ -343,8 +375,8 @@ func (k *Kademlia) Off(p OverlayConn) {
 // that has proximity order po or less as measured from the base
 // if base is nil, kademlia base address is used
 func (k *Kademlia) EachConn(base []byte, o int, f func(OverlayConn, int, bool) bool) {
-	k.lock.RLock()
```

Comment:
Why do the iterators have the `base []byte` param? If the base isn't kademlia base, should't merely the iterator from `pot/pot.go` be called directly?